exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -312,7 +312,8 @@ drill.exec: {
   memory: {
     operator: {
       max: 20000000000,
-      initial: 10000000
+      initial: 10000000,
+      output_batch_sizeinMB : 32
```

Comment:
Do we have two options when one would do? This is a config option. Below is a session/session option.